web_transport/sdk/api/owt/quic/web_transport_client_interface.h

```diff
@@ -31,18 +31,18 @@ class OWT_EXPORT QuicTransportClientInterface {
     // Called when a connection is failed.
```

Comment:
Can you provide more details in this interface document - define when OnConnectonFailed() will be invoked, (for example, when a connect fails, or when the underlying webtransport connection is no longer usable?)

Reply:
It fires when the state changed from connecting to failed.